packages/extensionmanager/src/companions.tsx

```diff
@@ -117,23 +117,23 @@ export function presentCompanions(
   }
   if (kernelCompanions.length > 0) {
     entries.push(
-      <p>
+      <p key={'kernal-companion'}>
         This package has indicated that it needs a corresponding package for the
         kernel.
       </p>
     );
-    for (let entry of kernelCompanions) {
+    for (let [index, entry] of kernelCompanions.entries()) {
       entries.push(
-        <p>
+        <p key={`compainion-${index}`}>
           The package
```

Comment:
The formatting of this line is a little off -- I think it would be fixed by removing the line break after `package`. Not your fault, but as long as you are here...? :)